dirsrvtests/tests/suites/clu/dsconf_config_test.py

```diff
@@ -53,6 +55,29 @@ def get_dsconf_base_cmd(topology):
             '-D', DN_DM, '-w', 'password']
 
 
+# Set PYTHONPATH without /usr/local to avoid mixing old CLI tools and new lib389
+def use_systemlib389():
+    savepath = None
+    if "PYTHONPATH" in os.environ:
+        savepath = os.environ["PYTHONPATH"]
+        srcpath = os.environ["PYTHONPATH"].split(":")
+    else:
+        srcpath = sys.path
+    newpath = []
+    for path in srcpath:
+        if not path.startswith("/usr/local"):
+            newpath.append(path)
+    os.environ["PYTHONPATH"] = ":".join(newpath)
+    return savepath
+
+
+def use_systemlib389_end(savepath):
```

Comment:
A bit suspicious: 
   This function will not be called if a test fails.

IMHO you should rather define use_systemlib389 as a function (or maybe module) fixture
 with a request parameter and add a finalizer top restore the environment
 
 BTW I am a bit surprised: savepath is undefined in use_systemlib389_end function ...